### ISSUE TITLE
Unnecessary commented code

### ISSUE THREAD

**Issue by skliarovartem** [vigilante95/Mongo-Express-Angular-NodeJs-MEAN-STACK-BLOG-with-auth#1]

**Comment by sanketdhoble** [comment]
@skliarovartem  What makes you think so?

**Comment by skliarovartem** [comment]
It is mix from different things, todo list and blog
Codestyle everywhere is different, many commented lines etc
Login/Registration controllers are headerController 
NodeJS: All api's in one file
And many small things
I would say from 10 it is 4

**Comment by sanketdhoble** [comment]
Thanks for reviewing the blog.
It was an attempt to learn NodeJs. I had earlier made a todo application by watching tutorials , and now I made this blog by changing previous todo code. 
I will remove the commented todo app code though.
Btw there are some features which I haven't seen in other blog examples l came across like
1)Post/update a blog by styling the content/article with an editor.
2)Most upvoted comments are placed at top (upvotes need to keep track of each user and should be unique).

**Comment by skliarovartem** [comment]
yes, good features. but you have to work on codestyle, tabs, spaces etc
